pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -166,7 +166,9 @@
       ],
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.3.0",
-        "4.0.2.0": "4.3.0"
+        "4.0.2.0": "4.3.0",
+        "4.0.1.0": "4.3.0",
```

Comment:
This looks wierd to me, why do we have 4.0.2 and 4.0.1 pointing to the same package?